specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.java.md

```diff
@@ -0,0 +1,63 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+azure-arm: true
+fluent: true
+namespace: com.microsoft.azure.management.containerservicefleet
```

Comment:
New `readme.java` for fleet, modified 
- `namespace` from `com.microsoft.azure.management.containerservice` to `com.microsoft.azure.management.containerservicefleet`
- `output-folder` from `$(azure-libraries-for-java-folder)/azure-mgmt-containerservice` to `$(azure-libraries-for-java-folder)/azure-mgmt-containerservicefleet`
- `title` from `ContainerServiceManagementClient` to `ContainerServiceFleetManagementClient`

Seems not able to keep namespace `containerservice` like golang.